### ISSUE TITLE
training tsn on kinetic400 is ok, but validating is very low. do you test your model on the kinetics400 data set

### ISSUE THREAD

**Issue by LiuChaoXD** [yjxiong/tsn-pytorch#114]

**Comment by LiuChaoXD** [comment]
I am training tsn model on kinetics400 data.
when the epoch is [30/100], I get the accuracy 30% on training set.
However, the accuracy of validate set is 5%.

**Comment by yojayc** [comment]
Did you solve the problem?

My training accuracy is also quite low

**Comment by LiuChaoXD** [comment]
> Did you solve the problem?
> 
> My training accuracy is also quite low

I think the problem is caused by the kinetics400 data set (probably).
I test the tsn model with their released trained weights on my kinetics400, unfortunately the accuracy is very low too.  I have no the original Kinetics400 data set, such that I don't know the problem is caused by the data sets or the model.

**Comment by cy-kui** [comment]
hello, i need the kinetics dataset, can you send me that? thank you! 2115996745@qq.com
